frontend/src/Onboarding/RedirectScreen.tsx

```diff
@@ -99,31 +70,25 @@ export const RedirectScreen: React.FC<Props> = ({ redirectUrl }) => {
               setIsLoading(false); // this update must come last, to make sure other state variables are correctly set before we redirect
             });
           })
-          .catch(e => console.log(e));
+          .catch(e => {
+            // TODO: Log error to some service like rollbar
+            console.log(e);
+            setIsError(true);
```

Comment:
We should also `setIsError(false)` at the start of this function, to handle the case of retrying after there was an error